monkestation/code/modules/storytellers/converted_events/_base_event.dm

```diff
@@ -220,18 +220,46 @@
 		else
 			candidates |= pick_n_take(possible_candidates)
 
+	var/failed_antags = 0
 	for(var/i in 1 to antag_count)
 		if(!length(candidates))
 			message_admins("A roleset event got fewer antags then its antag_count and may not function correctly.")
 			break
 
 		var/mob/candidate = pick_n_take(candidates)
+
+		if(prompted_picking)
+			var/choice = tgui_input_list(candidate, "Would you like to be a [cast_control.name]?", "Antag Selection", list("Yes", "No"), "No", 7 SECONDS)
```

Comment:
This should be `tgui_alert` instead of `tgui_input_list` due to only having two options, also prompted_picked is already handled on lines 218 and 219. It uses poll_candidates due to it handling some checks.